techmatters/terraso-mobile-client
# feat: implement soil color algorithm

## Description
Ports the soil color algorithm code to our codebase. Implements the image picker/cropping workflow to create the inputs to the soil color algorithm, and saves the output. Also finishes filling out the assets for the color guide, and properly displays munsell colors.

Note: most of the added lines are from a giant precomputed table of munsell color values, so the PR is a lot smaller than it looks.

### Checklist

### Related Issues
Next step for #129. See that issue for a list of remaining work for the soil color screen.

Depends on https://github.com/techmatters/terraso-client-shared/pull/289.

### Verification steps
User should be able to use the color algorithm to crop a photo from their library and get an estimated munsell color. The accuracy of the estimate can vary wildly depending on the lighting conditions in the photo. See here for some example images you can use to test: https://drive.google.com/drive/folders/1gIYC6yxpHXJ962QkFPx2TuqFG9Z20GWU
